coreteam.py

```diff
@@ -0,0 +1,71 @@
+##
+# List of PrestaShop employees or subcontractors
+#
+
+CORE_TEAM = (
+    'PierreRambaud',      # Pierre R.
+    'LittleBigDev',       # Bastien B.
+    'Quetzacoalt91',      # Thomas N.
+    'azouz-jribi',        # Azouz Jribi (Sifast)
+    'eternoendless',      # Pablo B.
+    'fatmaBouchekoua',    # Fatma B. (Sifast)
+    'jocel1',             # Jocelyn F.
+    'mickaelandrieu',     # Mickael A.
+    'rGaillard',          # Remi G.
+    'tomlev',             # Thomas L.
+    'toutantic',          # Aurelien
+    'slorenzini',         # Sarah L.
+    'LouiseBonnard',      # Louise B.
+    'ttoine',             # Antoine T.
+    'colinegin',          # Coline G.
+    'marionf',            # Marion F.
+    'matks',              # Matthieu F.
+    'khouloudbelguith',   # Khouloud B. (Sifast)
+    'fouratachour',       # Fourat A.
+    'sLorenzini',         # Sarah L.
+    'apacios',            # Adrien P.
+    'jolelievre',         # Jonathan L.
+    'clementdaubeuf',     # Clement D.
+    'MrBaiame',           # Alexis V.
+    'Joukz',              # Julien G.
+    'david-piatek',       # David P.
+    'yelnatss',           # Marvin S.
+    'CaptainYouz',        # Anas M.
+    'Matt75',             # Matthias R.
+    'SimonGrn',           # Simon G.
+    'mbadrani',           # Mehdi B.
+    'boubkerbribri',      # Boubker B.
+    'matthieu-rolland',   # Matthieu R.
+    'Progi1984',          # Franck L.
+
+)
+
+##
+# List of PrestaShop project repositories
+#
+
+PROJECTS = {
+    'prestashop-specs' : 'PrestaShop Specifications',
+    'docs' : 'Changes in developer documentation',
+    'TranslationToolsBundle' : 'TranslationTools Bundle',
+    'QANightlyResults' : 'QA nightly results',
+    'php-coding-standards' : 'PHP Coding Standards',
+    'ps_facetedsearch' : 'Faceted search module',
+    'gamification' : 'Gamification module',
+    'autoupgrade' : 'Auto Upgrade module',
+    'classic-rocket' : 'Classic-rocket theme',
+    'prestashop-ui-kit' : 'Prestashop UI Kit',
```

Comment:
Aren't we missing like 80 repos?

Reply:
The goal of this list is to speed up the polishing work done after the generation as these are the repositories "most often updated". I dont think we should put all the repositories in it, only the ones most often updated. This list can grow over time.